blathering-barnacles/Bangazon-3
# Ah customer filter

## Link to Ticket
Closes #1 

## Description of Proposed Changes

- 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <feature_branch>
```
http://localhost:8000/api/v1/customers/?q=andy
only andy should show up
you can type partial names and names with those letters should also show up

## Impacted Areas in Application

List general components of the application that this PR will affect:

* 

## Mentions @username

Tag users that need to review this code
